### DISCUSSION TITLE
Notcurses 3.0 / libnotcurses.so.3

### DISCUSSION THREAD

**Discussion by dankamongmen** [dankamongmen/notcurses#1949] [General]
We are now 66% done with the states work items for Notcurses 3, having resolved 175 of 263. Eighty-eight remain open. I'm sure a good number will be opened before we're done.

I'm not sure I see a Notcurses 4, definitely not in the immediate horizon. Notcurses 3 will include all functionality I knew I wanted when I started this project in November of 2019. The API will be fairly well proven in numerous scenarios.

Does anyone else see any groundbreaking additions that would possibly justify a Notcurses IV, aside from the unceasing need for new [triumphant videos](https://youtu.be/SWIHaXsNyko)?

**Comment by WSLUser** [comment]
I think providing the ncurses shim would need to be a 3.0 or possibly a 4.0 item.  I also believe that as terminals evolve along with everything else in the world, this project will need to be continued to be maintained and improved as changes occur. I do hope that you will continue to do so once 3.0 is released and not abandon the project like other projects have had happen.
